Info-Orbs/src/widgets/webDataModel.cpp

```diff
@@ -70,6 +103,59 @@ void WebDataModel::setBackgroundColor(String background) {
 bool WebDataModel::isChanged() {
     return m_changed;
 }
+
 void WebDataModel::setChangedStatus(bool changed) {
     m_changed = changed;
 }
+
+void WebDataModel::parseData(const JsonObject& doc, int32_t defaultColor, int32_t defaultBackground) {
+    if (doc.containsKey("label")) {
+        setLabel(doc["label"].as<String>());
+    }
+    if (doc.containsKey("data")) {
+        setData(doc["data"].as<String>(), defaultColor, defaultBackground);
+    }
+    if (doc.containsKey("color")) {
+        setDataColor(doc["color"].as<String>());
```

Comment:
I don't think you need the `.as<String>()` for this library.

Reply:
yes, we do `doc["color"]` is a JsonObject so we need `.as<String>()` to convert it to a string. I suppose we could pass the JsonObject and cast it in the function but I don't see any benefits to that.

Reply:
The documentation on https://arduinojson.org/ claims:
> Implicit or Explicit Casts
> ArduinoJson supports two coding styles, with implicit or explicit casts
> 
> Implicit cast:
>
> ```
> const char* city = doc["city"];
> float temp = doc["temperature"];
> ```
> Explicit cast:
>
> ```
> auto city = doc["city"].as<const char*>();
> auto temp = doc["temperature"].as<float>();
> ```

Reply:
I think that would work if we copied it to a variable but using it as an argument for a function call doesn't seem to be enough context for the implicit casts.